crates/crypto/src/key/public_key.rs

```diff
@@ -39,7 +38,7 @@ mod tests {
         let signature = private_key.sign(payload).unwrap();
 
         let public_key = private_key.to_public();
-        let verification_warnings = public_key.verify(&payload, &signature).unwrap();
+        let verification_warnings = public_key.verify(payload, &signature).unwrap();
```

Comment:
What provoked this change? Why is `signature` passed by reference but `payload` isn't?

Reply:
This is the error https://rust-lang.github.io/rust-clippy/master/index.html#/needless_borrow

Reply:
BTW this was the result of running `cargo clippy --fix`

Reply:
Oh I see, because `payload` was already initialized by reference whereas `signature` was not